capstone/src/main/resources/static/script.js

```diff
@@ -425,6 +437,31 @@ function createCalendar() {
     dateDiv.appendChild(dateDisplay);
     calendar.append(dateDiv);
   }
+}
+/*
+ * If element is selected, border will change from white to green
+ * @param element id
+ */
+function toggleBorderSelection(elementId) {
+  var element = document.getElementById(elementId);
+  if (element.nodeName == 'BUTTON') {
+    element.classList.toggle('selected');
+  } else if (element.nodeName == 'SELECT') {
+    if (element.value != '') {
+      element.classList.add('selected');
+    } else {
+      element.classList.remove('selected');
+    }
+  }
+}
 
+/*
+ * Reviews are temp on event listing so remove to see finalized event search page
+ */
+function removeReviews() {
+  const reviews = document.getElementsByClassName('reviews');
+  while(reviews.length > 0){
+    reviews[0].parentNode.removeChild(reviews[0]);
+  }
```

Comment:
I like the layout of the events once the reviews are removed. Maybe this is a button we should add to temporarily hide reviews if the user does not care about them.

Reply:
Same! In the final project, I think the reviews will be attached to the event's detail page instead of the event search. Whenever those are created I'll just move the review element.